target/linux/tegra/image/Makefile

```diff
@@ -21,7 +21,7 @@ define Build/tegra-sdcard
 		-e 's#@KERNEL@#$(KERNEL_NAME)#g' \
 		$(BOOT_SCRIPT) > $@-boot.scr
 	mkimage -A arm -O linux -T script -C none -a 0 -e 0 \
-		-n '$(DEVICE_TITLE) OpenWrt bootscript' \
+		-n '$(DEVICE_VENDOR) $(DEVICE_MODEL) OpenWrt bootscript' \
```

Comment:
I do not think this is necessary at all, since DEVICE_TITLE will still be set based on the other two/three variables. With the possibility of having DEVICE_VARIANT set, the original solution is even better than the new one.
So, this PR seems unnecessary to me, or am I overlooking something?

Reply:
No, you are right. This PR was about to clear up this issue.